### DISCUSSION TITLE
Bei Migration auf Raspberrymatic werden keine HM-IP Geräte übernommen

### DISCUSSION THREAD

**Discussion by mkollermann** [OpenCCU/OpenCCU#1135] [🇩🇪 Fragen&Antworten] [answered]
Liebe Alle, lieber Jens,

Vielleicht gibt es hierzu schon einen Thread, ich habe aber keinen gefunden.
Ich hätte versucht von einer CCU3 auf die Raspberrymatic umzusteigen.

Dazu habe ich:

1) Die neueste Version der Raspberrymatic auf der SDHC Karte installiert
2) Das Backup der CCU3 (selbe Firmwareversion) eingespielt

Leider scheinen aber sämtliche Homematic IP Geräte, egal ob Wired oder Wireless nicht in der Geräteliste auf.
Hat hier jemand einen Vorschlag was man ausprobieren kann ?

LG
Markus

**Comment by blackbasket** [comment]
Hallo Markus,

ich kann das Verhalten aktuell nicht nachvollziehen. Ich habe Samstag (frische Installation auf neuer SD-Karte) und Dienstag ("Update" einer CCU3) zwei Varianten zum Umstieg genutzt und beide CCUs laufen nun erfolgreich auf RM.

Hast du schon mal getestet, ob du das Backup auf eine Original-CCU-FW zurückspielen kannst? Nicht, dass dein Backup generell ein Problem hat.

LG,
Marcel

**Comment by mkollermann** [comment]
Hallo Marcel,

Nein, das habe ich noch nicht versucht….Möchte ich ehrlich gesagt auch nicht versuchen 😊
Ich habe mir alles parallel aufgebaut um im Notfall wieder einfach zurücksteigen zu können…
Ich habe schon 2 mal die Karte geflasht und unterschiedliche Backups auf das neue System restored…
Hast es auch auf einem Raspberry 4 versucht ?
Die HM-Geräte funktionieren alle einwandfrei, sogar die Verbindung zu Cloudmatic wurde mit übernommen.
Aber das DRAP und die weiteren Wireless HM-IP Komponenten sind halt nicht da…

LG
Markus

**Reply by blackbasket** [reply]
Hallo Markus,

nein, da ich bisher überall CCU3s nutze, sind es alles RPi3.

Blöde Frage: Du hast die alte CCU aber abgeschaltet, während du die neue testest?

LG,
Marcel

PS: ich würde die Signatur löschen, wenn du per Mail antwortest ;)

**Comment by mkollermann** [comment]
Hallo Marcel,

Ja, habe ich natürlich abgeschalten und vom Netzwerk getrennt… 😊…

LG
Markus

**Reply by blackbasket** [reply]
Schade ;) dann habe ich aktuell keine Idee...

**Reply by jens-maus** [reply]
@mkollermann BTW: Bitte es unterlassen auf die GitHub Emails direkt (vor allem mit sinnlosem FullQuote) im Email-Client zu antworten, denn dann steht dieser Müll auch auf der GitHub Seite für jeden einsehbar:

> LG Markus mit freundlichen Grüßen XXXXX Von: blackbasket <notifications@github.com> Gesendet: Donnerstag, 18. Februar 2021 11:21 [...]

Das beste ist einfach dem Link zu folgen und direkt unter GitHub zu antworten und seinen Email-Client für sowas nicht zu nutzen!

**Comment by mkollermann** [comment]
Ich habe noch eine Frage: Lt. Jens (Oder bist das eh Du Marcel?- kenn mich nicht mehr aus 😊) sollte ich die 62HMServer datei editieren.

Leider kann ich die nicht wegschreiben, obwohl ich als root eingelogt bin. Wie kann ich das machen ? Any Idea ?

**Reply by jens-maus** [reply]
> Ich habe noch eine Frage: Lt. Jens (Oder bist das eh Du Marcel?- kenn mich nicht mehr aus 😊) sollte ich die 62HMServer datei editieren. Leider kann ich die nicht wegschreiben, obwohl ich als root eingelogt bin. Wie kann ich das machen ? Any Idea 

Entscheide dich erst einmal wo du deine Frage in Zukunft stellst. Doppelposts auch Forenübergreifend gehören sich nicht!

Siehe: https://homematic-forum.de/forum/viewtopic.php?f=65&t=65656&p=645543#p645543

**Reply by mkollermann** [reply]
Ja stimmt. Sorry.

**Comment by jens-maus** [comment, marked as answer]
Die Lösung dieses Anliegens hier sollte übrigens (für andere auffindbar) hier zu finden sein:
https://homematic-forum.de/forum/viewtopic.php?p=645509#p645509
